OpenRA.Game/Server/Server.cs

```diff
@@ -82,7 +82,7 @@ public sealed class Server
 		static readonly string CustomRules = "custom-rules";
 
 		[TranslationReference]
-		static readonly string BotsDisabled = "bots-disabled";
+		static readonly string BotsDisabled = "no-bots";
```

Comment:
Why the rename? I feel this makes it less clear.

Reply:
To avoid a collision.